infrastructure/pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+name = "infrastructure"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
```

Comment:
Not necessary in my experience but I won't hold up the PR on this.

Reply:
fix downstream